GNUmakefile

```diff
@@ -295,6 +296,15 @@ exporter-image: exporter
 	@cd buildscripts/exporter && sudo docker build -t openebs/m-exporter:${IMAGE_TAG} --build-arg BUILD_DATE=${BUILD_DATE} .
 	@rm buildscripts/exporter/${EXPORTER}
 
+webhook-image:
```

Comment:
this is not called in any of build scripts.. this can be taken in next PR after discussion about merging this with current binary

Reply:
Yes .. this is intentional, thought of adding it later.

Reply:
Added